### DISCUSSION TITLE
Expedition 4 - End Game Reward - Hungering Worm Companion Egg

### DISCUSSION THREAD

**Discussion by Alanray64** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#50] [General]
Just finished this off today and when I went to collect the rewards I only received two, think it was the Titans Skull Decoration and a flag or something. I went looking for the egg and I am unable to see it anywhere. I do have a pet called Egaui-Egauy which looks like a sandworm but I am sure I got that elsewhere. When I view the redeemed rewards there is a Sandworm egg, Titanic Spawn
![image](https://github.com/user-attachments/assets/9847bb25-5234-440b-961c-2e4db1de341b)

This is the only worm I see. The reward speaks of a Hungering Worm?

A bug or?

Thanks for all these buy the way, they have been a lot of fun to do.

**Comment by cwmonkey** [comment]
Since you can't transfer eggs back to your main save I don't really pay much attention to them. That said, I feel like I remember getting an egg on that expedition (I did it a few days ago). Worth noting you also get that egg on the Omega expedition if you're really wanting to get it (and a bunch of other eggs).

I'd go test it but I dread having to get all the way to the end again, lol. I'll make a note to check this eventually.

**Reply by Alanray64** [reply]
yeh I hear you. I know I received a worm egg on another expedition but as they were titled differently I had assumed it was another unique item. The past expeditions allowed collecting the reward egg from the vendor (did try transferring but they were greyed out) however this time, I only received two rewards so was a bit unsure as I had thought each phase completion would give something that at the end would be added to the original save.

No great issue, just brought it up and was wondering the whys behind :)

Thanks again.

**Reply by cwmonkey** [reply]
The Omega egg and the Emergence eggs are the same, so if you collected it after Omega it wouldn't be available after Emergence.

Regardless of what happened, you can always use a save editor to get one. The [GoatFungus editor](https://github.com/goatfungus/NMSSaveEditor) has it and is easy to find:

![image](https://github.com/user-attachments/assets/d5661a7f-5d65-4ad8-8590-9a4780ae7360)

Either way I'll double-check the reward eventually, since they do break sometimes.
